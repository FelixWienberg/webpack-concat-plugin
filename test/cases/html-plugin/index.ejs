<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <title>Webpack App</title>
  </head>
  <body>
  <script src="<%= htmlWebpackPlugin.files.webpackConcat.file %>"></script>
  </body>
</html>
