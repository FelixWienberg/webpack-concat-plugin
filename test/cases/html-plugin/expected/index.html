<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <title>Webpack App</title>
  </head>
  <body>
  <script src="file.0ff883d05f62ecabac00.js"></script>
  <script src="main.js"></script></body>
</html>
